pkg/bootstrap/user/userprofile.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package user
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/trustbloc/edge-core/pkg/storage"
+)
+
+// Profile is the user's bootstrap profile.
+type Profile struct {
+	Sub               string
+	SDSPrimaryVaultID string
+	KeyStoreIDs       []string
+}
+
+// ProfileStore is the user Profile CRUD API.
+type ProfileStore struct {
+	s storage.Store
+}
+
+// NewStore returns a new ProfileStore.
+func NewStore(s storage.Store) *ProfileStore {
+	return &ProfileStore{s: s}
+}
+
+// Save saves the user Profile.
+func (ps *ProfileStore) Save(p *Profile) error {
+	bits, err := json.Marshal(p)
+
+	if err != nil {
+		return fmt.Errorf("failed to marshal user profile : %w", err)
```

Comment:
extra space before `:`

(also in other error messages in this PR)

(this is a nitpick :) )